modules/claims_api/app/controllers/claims_api/v2/veteran_identifier_controller.rb

```diff
@@ -44,6 +45,34 @@ def validate_birthdate!(date_str)
       rescue ArgumentError
         raise ::Common::Exceptions::InvalidFieldValue.new('birthdate', date_str)
       end
+
+      def find_veteran(params)
+        test_veteran_data = {
+          id: ICN_FOR_TEST_USER,
+          ssn: '796130115',
+          firstName: 'Tamara',
+          lastName: 'Ellis',
+          birthdate: '1967-06-19'
+        }
+
+        unless params[:ssn] == test_veteran_data[:ssn]
```

Comment:
more out of curiosity, why do `casecmp?` for the names down below but not here?

Reply:
Because this is a social security number.
There shouldn't be any case sensitivity issues with digits.  (unless I'm missing something, haha 😂 )